src/components/SUSHIControls.js

```diff
@@ -41,24 +41,33 @@ const useStyles = makeStyles((theme) => ({
   }
 }));
 
-const theme = createMuiTheme({
-  typography: {
-    fontFamily: 'Open Sans'
-  }
-});
-
```

Comment:
Is there a reason you took this out? I think this was added in during your stying PR and it makes the font on the button match the one in the top bar.